README.md

```diff
@@ -55,6 +77,7 @@ pip install -r requirements.txt
 | `DB_PASSWORD` | Postgres password | |
 | `DB_HOST` | Hostname of postgres server | `127.0.0.1` |
 | `DB_PORT` | Postgres port | `5432` |
+| `ENABLED` | Feature flags (see base.py) | |
```

Comment:
Good one and I like that you point to the file where these are 👍 - I would probably change the name in the README to `ENABLE_*` so it's even clearer that you're referring to the environment variables starting with `ENABLE_`.